libs/velo-external-db-core/src/service/schema.spec.ts

```diff
@@ -31,36 +33,39 @@ describe('Schema Service', () => {
             await expect( env.schemaService.list([]) ).resolves.toEqual(collectionsListFor(ctx.dbsWithIdColumn))
         })
 
-        test('create new collection without fields', async() => {
+        test('create new collection without fields should throw', async() => {
             driver.givenAllSchemaOperations()
             driver.expectCreateOf(ctx.collectionName)
             schema.expectSchemaRefresh()
 
-            await expect(env.schemaService.create({ id: ctx.collectionName, fields: [] })).resolves.toEqual({
-                collection: { id: ctx.collectionName, fields: [] } 
-            })
+            await expect(env.schemaService.create({ id: ctx.collectionName, fields: [] })).rejects.toThrow(errors.InvalidRequest)
+        })
+
+        test('create new collection without _id field should throw', async() => {
+            driver.givenAllSchemaOperations()
+            schema.expectSchemaRefresh()
+            driver.expectCreateWithFieldsOf(ctx.collectionName, [ ctx.column ])
+
+            await expect(env.schemaService.create({ id: ctx.collectionName, fields: InputFieldsToWixFormatFields([ ctx.column ]) })).rejects.toThrow(errors.InvalidRequest)
         })
 
         test('create new collection with fields', async() => {
-            const fields = [{
-                key: ctx.column.name,
-                type: fieldTypeToWixDataEnum(ctx.column.type),
-            }]
+            const fields = [
+                { key: PrimaryKeyFieldName, type: VeloFieldTypeEnum.text },
+                InputFieldToWixFormatField(ctx.column),
+            ]
+
             driver.givenAllSchemaOperations()
             schema.expectSchemaRefresh()            
-            driver.expectCreateWithFieldsOf(ctx.collectionName, fields)
+            driver.expectCreateWithFieldsOf(ctx.collectionName, WixFormatFieldsToInputFields(fields))
```

Comment:
lets add here test that we are calling the schema with the right columns.
` expect(driver.schemaProvider.create).toBeCalledWith(collectionName, columnsAfterTranslation)`